src/Elcodi/Component/Language/Twig/LanguageExtension.php

```diff
@@ -19,9 +19,12 @@
 
 use Twig_Extension;
 use Twig_Extension_GlobalsInterface;
-
```

Comment:
LOL, yes, sorry :/
Let's merge this and will be solved in next PR, agree?
BTW, we need to find why php-cs-fixer is ignoring the configuration of php_cs file (is explicit that this check is disabled...)